networking/hub-default.json

```diff
@@ -363,9 +363,101 @@
                 "mode": "Parallel"
             }
         },
+        {
+            "type": "Microsoft.Network/firewallPolicies",
+            "apiVersion": "2020-11-01",
+            "name": "[variables('hubFwName')]",
+            "location": "[parameters('location')]",
+            "properties": {
+                "sku": {
+                    "tier": "Standard"
+                },
+                "threatIntelMode": "Deny",
+                "threatIntelWhitelist": {
+                    "ipAddresses": []
+                },
+                "dnsSettings": {
+                    "servers": [],
+                    "enableProxy": true
+                }
+            },
+            "resources": [
+                {
+                    "type": "Microsoft.Network/firewallPolicies/ruleCollectionGroups",
+                    "apiVersion": "2020-11-01",
+                    "name": "[concat(variables('hubFwName'), '/DefaultDnatRuleCollectionGroup')]",
+                    "location": "[parameters('location')]",
+                    "dependsOn": [
+                        "[variables('hubFwName')]"
+                    ],
+                    "properties": {
+                        "priority": 100,
+                        "ruleCollections": []
+                    }
+                },
+                {
+                    "type": "Microsoft.Network/firewallPolicies/ruleCollectionGroups",
+                    "apiVersion": "2020-11-01",
+                    "name": "[concat(variables('hubFwName'), '/DefaultApplicationRuleCollectionGroup')]",
+                    "location":"[parameters('location')]",
+                    "dependsOn": [
+                        "[variables('hubFwName')]",
+                        "[resourceId('Microsoft.Network/firewallPolicies/ruleCollectionGroups', variables('hubFwName'), 'DefaultDnatRuleCollectionGroup')]"
```

Comment:
This depends on itself?

Reply:
It not it self. It is the same type, not the same resource.
You get a deploy error if you deploy all  Microsoft.Network/firewallPolicies/ruleCollectionGroups deploy together.
The error  was "there is another update operation executing, bad request"
The only solution solution was a dependency between them. It was the biggest issue of this task. 
The first ruleCollectionGroups  doesn't any dependency
the second depends on the first  ruleCollectionGroups 
the third depends on the second ruleCollectionGroups .

Reply:
Ah,  you're right, I read that poorly, closing this one comment.

Reply:
![image](https://user-images.githubusercontent.com/62260324/115920347-73ad2c80-a450-11eb-88d9-2cd745e799e5.png)
the error deleting the dependency 
{
    "status": "Failed",
    "error": {
        "code": "FirewallPolicyRuleCollectionGroupUpdateNotAllowed",
        "message": "Rule Collection Group DefaultDnatRuleCollectionGroup can not be updated because Parent Firewall Policy fw-eastus2 is in Updating state from previous operation ID : f21d4ad1-1db0-44d2-8c0e-ba2b0cfa8b15."
    }
}

Reply:
I'm not sure what this means (as far as this comment thread)

Reply:
the error that you get deleting this comment dependency mentioned before 
```
The first ruleCollectionGroups doesn't any dependency
the second depends on the first ruleCollectionGroups
the third depends on the second ruleCollectionGroups .
```

Reply:
deleting a comment dependency?  I'm not sure I follow.  Sorry.  Feel free to ping me on teams, I must be missing something.

Reply:
OH, I get it now. I misread something you wrote -- interpreted it another way.  Marking this resolved.